app/models/user.rb

```diff
@@ -8,15 +8,14 @@ class User < ActiveRecord::Base
 
   accepts_nested_attributes_for :authentications
 
-  validates :password, length: { minimum: 6, message: "Длина пароля должна быть не меньше 6 символов" },
-            if: -> { new_record? || changes["password"] }
-  validates :password, confirmation: { message: "Пароли не совпадают" },
-            if: -> { new_record? || changes["password"] }
-  validates :password_confirmation, presence: { message: "Поле не может быть пустым" },
-            if: -> { new_record? || changes["password"] }
+  validates :email, uniqueness: { message: 'Такой email уже существует' },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.